### PR TITLE
build doc - build geonetwork first

### PR DESCRIPTION
**Pull request by jdenisgiguere**
When compiling geOrchestra for the first time, Geonetwork must be build first.

### CODE REVIEW

File: doc/build.md [hunk 1]
```diff
@@ -79,14 +79,25 @@ data (using a batch insert for instance).
...
 cd ~/georchestra
 export MAVEN_OPTS="-XX:MaxPermSize=512M"
-./mvn -Dmaven.test.skip=true -Dserver=myprofile clean install
 ```
 
-Note: this will build **all modules** (except GeoFence).
+Build Geonetwork
+```
+cd geonetwork
+./mvn -DskipTests clean install
```
Comment (fvanderbiest) [inline doc/build.md]: rather `../mvn` ?

File: doc/build.md [hunk 2]
```diff
@@ -79,14 +79,25 @@ data (using a batch insert for instance).
...
+cd ..
+```
+
+Build **all modules** (except GeoFence).
+```
+./mvn -Dmaven.test.skip=true-Dserver=myprofile clean install
```
Comment (fvanderbiest) [inline doc/build.md]: missing space between `-Dmaven.test.skip=true` and `-Dserver=myprofile`

File: doc/build.md (hunk 1 above)
Comment (jdenisgiguere) [inline doc/build.md]: You're right! Fixed in cc3c4ed6ca7e02f605e62190866bf134fc390a89.

**Comment by jdenisgiguere** [comment]
Thank you @fvanderbiest  for rigorous review. I hope this will make life easier for the newcomers.

**Comment by fvanderbiest** [comment]
Looks good, thanks Jean Denis.

**Pull request merged**